stories/Components/Navigationcomponents/Mainnavigation/MobileNav/MobileNav.jsx

```diff
@@ -79,14 +73,12 @@ const MobileNav = ({
           </div>
         </div>
 
-        <div className="cell small-12 mob-sub-lang">
-          <div className="back-nav">
-            <span>
-              <Icons src={Iconsback} alt="icon-back" />
-              {' '}
-              {backcaption.detail}
-            </span>
-          </div>
+        <div className="cell mob-sub-lang">
+          <span className="back-nav">
+            <Icons src={Iconsback} alt="icon-back" />
```

Comment:
@Ashish-srij Change the above to a back `button` and **not** `span` since it has click action associated with it. Also change `alt` here to `back button`.

Reply:
@shishirphadke Done.